doc/spectrographs/soar_goodman.rst

```diff
@@ -0,0 +1,15 @@
+************
+SOAR Goodman
+************
+
+Overview
+========
+
+This file summarizes several instrument specific
+items for the SOAR/Goodman spectrograph. Note that
+there are two ways to download your data from the
+archive:
+(1) in .fits format; and
```

Comment:
If you intended this to appear as a numbered list, there needs to be a newline before the `(1)`. And then one after the the `(2)` and the `PypeIt will only work...` paragraph.

Reply:
Fixed - thanks!